src/options.js

```diff
@@ -0,0 +1,97 @@
+"use strict";
+
+const path = require('path');
+
+// These datasets can be exported
+const allDatasets = [
+  "categories",
+  "occasiondata",
+  "occasions",
+  "parties",
+  "theses_categories"
+];
+
+const DEFAULT_DATA_DIR = path.join("..", "export");
+
+const commandLineOptions = [
+  { name: 'verbose', alias: 'v', type: Boolean, defaultValue: false },
+  { name: 'datasets', type: String, multiple: true, defaultOption: true,
+    defaultValue: allDatasets },
+  { name: 'csv', type:Boolean, defaultValue: false },
+  { name: 'help', alias: 'h', type: Boolean },
+  { name: 'target', alias: 't', type: String, defaultValue: DEFAULT_DATA_DIR }
+];
+
+const optionDescriptions = [
+{
+  name: 'datasets',
+  description: 'Names of datasets to export. See section `Datasets` below.',
+  multiple: true,
+  defaultOption: true,
+  typeLabel: '[underline]{name} ...'
+},
+{
+  name: 'verbose',
+  description: 'Enable verbose output',
+  alias: 'v',
+  type: Boolean
+},
+{
+  name: 'csv',
+  description: 'Export all occastions and theses as csv',
+  type: Boolean
+},
+{
+  name: 'target',
+  alias: 't',
+  description: 'Target directory for saving files. Default is `../export/`.',
```

Comment:
won't work, if target directory (e.g. default export) is not already created

Reply:
fixed